lib/capistrano/tasks/wp.cap

```diff
@@ -12,6 +24,46 @@ namespace :wp do
 
   namespace :setup do
 
+    # Generates a random password and stores it as :wp_pass
+    task :generate_password do
+      o = [('a'..'z'), ('A'..'Z')].map { |i| i.to_a }.flatten
+      password = (0...18).map { o[rand(o.length)] }.join
+      set :wp_pass, password
+    end
+
+    # Installs WordPress
+    task :install
+      config = Config.new
```

Comment:
It seems this line here is not happy with arguments not being passed. 
With no argument passed it reads `ArgumentError: wrong number of arguments (given 0, expected 1)`
with `(options)` passed as the argument it reads `NameError: undefined local variable or method `options' for main:Object`

Not sure how to resolve this since I'm not familiar with Ruby classes and pulling dynamic variables.